### PR TITLE
i18n(zh-CN): add translation for `modified-time.mdx`

### PR DESCRIPTION
**Pull request by Genteure**
#### Description (required)

Add translation for `modified-time.mdx`.
Source: https://github.com/withastro/docs/blob/main/src/content/docs/en/recipes/modified-time.mdx

Related to #5039.

### CODE REVIEW

File: src/content/docs/zh-cn/recipes/modified-time.mdx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+         const result = execSync(`git log -1 --pretty="format:%cI" ${filepath}`);
+         file.data.astro.frontmatter.lastModified = result.toString();
+       };
+     }
+     ```
+		<details>
+		<summary>使用文件系统而不是 Git</summary>
+
+		 虽然使用 Git 是获取文件的最后修改时间的推荐方式，但使用文件系统的修改时间也是可能的。
+     这个插件使用 `statSync` 获取文件的 `mtime` (修改时间)，然后将时间戳添加到文件的 frontmatter 中。
```
Comment (100gle) [inline src/content/docs/zh-cn/recipes/modified-time.mdx:55]:
```suggestion
     这个插件使用 `statSync` 获取文件的 ISO 8601 格式的 `mtime` (修改时间)，然后将时间戳添加到文件的 frontmatter 中。
```

**Review by 100gle — CHANGES_REQUESTED** [review]
漏了点原文的表述

**Review by 100gle — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🙌

**Pull request merged**